config/Dockerfiles/JenkinsfileContainer

```diff
@@ -11,7 +11,7 @@ openshiftProject = "continuous-infra"
 DOCKER_REPO_URL = env.DOCKER_REPO_URL ?: '172.30.254.79:5000'
 
 // Defaults for SCM operations
-env.ghOrg = 'herlo'
+env.ghOrg = '14rcole'
```

Comment:
This should really be `CentOS-PaaS-SIG`, but this environment variable is used to pull a version of the `ci-pipeline` repo that contains a branch called `automate_lp`.  Should we try to get that branch pushed to `CentOS-PaaS-SIG/ci-pipeline`?

Reply:
You could check that branch out yourself and diff to see what is needed and then add that to this pr if missing.

Reply:
it's a different project.  I assumed there was a reason that the changes were never merged.  Who are the maintainers for ci-pipeline?

Reply:
@14rcole 
I see @johnbieren @p3ck @arilivigni As top contributors.

Reply:
@samvarankashyap that is the repo our entire team used to create the atomic pipeline. You might remember creating the linchpin-libvirt stage as part of that workflow.

Reply:
@14rcole, In that case, I believe you might have access to ci-pipeline. 
could you push automate_lp branch as a **separate branch** of ci-pipeline? and use that branch as a reference in Jenkinsfile.

Reply:
@samvarankashyap If you push a separate branch to ci-pipeline, you would have to maintain it. Why wouldn't you just want to inspect the differences between the branches and merge whatever is necessary, like Bill said? I would implore you to look at https://github.com/CentOS-PaaS-SIG/ci-pipeline/compare/master...herlo:automate_lp instead of pushing a separate branch to the repo that will eventually become stale, as Bill said.

Reply:
@johnbieren I'm happy to do that, but I assumed there was a reason it was left separate

Reply:
https://github.com/CentOS-PaaS-SIG/ci-pipeline/pull/804